Backend-API/src/main/java/com/avangers/backendapi/services/UserService.java

```diff
@@ -6,4 +6,6 @@
 
 public interface UserService extends UserDetailsService {
   RegisterUserResponseDTO addUser(RegisterUserRequestDTO registerUserRequestDTO);
+  RegisterUserResponseDTO updateUser(Long userId, RegisterUserRequestDTO updateUserRequestDTO);
```

Comment:
maybe it is better to use different DTO, not register

Reply:
```suggestion
 UpdateUserResponseDTO updateUser(Long userId, UpdateUserRequestDTO updateUserRequestDTO);
```

maybe < ?